src/background/account/Account.ts

```diff
@@ -46,27 +61,29 @@ export class Account extends EventEmitter<AccountEvents> {
   isPendingNewUser: boolean;
 
   private static async writeCurrentUser(user: User) {
-    await browserStorage.set(currentUserKey, user);
+    await BrowserStorage.set(currentUserKey, user);
```

Comment:
`User` data contains `salt`:
```
export interface User {
  id: string;
  salt: string;
}
```
I'm not sure how risky it is to store the `salt` in browser storage or if we can keep user data in session storage

Reply:
Encrypted storage cannot be unlocked without it, this is part of necessary data

Reply:
Is it possible to make the `salt` a hash of the password instead of using `crypto.getRandomValues(new Uint8Array(32))`, so that we wouldn’t need to store it in browser storage (it could be stored in the session store).
But I’m still not sure if this is a good idea